exo-go/src/terraform_file_helpers/template_builder.go

```diff
@@ -0,0 +1,48 @@
+package terraformFileHelpers
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/hoisie/mustache"
+	"github.com/pkg/errors"
+)
+
+// RenderTemplates renders a Terraform template
+func RenderTemplates(templateName string, varsMap map[string]string) (string, error) {
+	template, err := getTemplate(templateName)
+	if err != nil {
+		return "", err
+	}
+	return mustache.Render(template, varsMap), nil
+}
+
+// WriteTerraformFile writes the main Terraform file to the path: cwd/terraform/main.tf
+func WriteTerraformFile(data string) error {
+	cwd, err := os.Getwd()
```

Comment:
Please pass in the cwd / appDir in as a parameter. Lets just get the working directory once in the command and pass it to the helper functions as needed. This also makes things easier to test.